lib/icingadb/redisconnection.cpp

```diff
@@ -413,12 +422,14 @@ void RedisConnection::ReadLoop(asio::yield_context& yc)
 								throw;
 							} catch (...) {
 								promise.set_exception(std::current_exception());
-
-								continue;
+								break;
 							}
 						}
 
-						promise.set_value(std::move(replies));
+						try {
+							promise.set_value(std::move(replies));
+						} catch (const std::future_error&) {
+						}
```

Comment:
Please at least add a comment why that exception is ignored here (having some `bool failed` would be somewhat more self-documenting though)